### ISSUE TITLE
Refresh when opening RD windows?

### ISSUE THREAD

**Issue by AndreasMatthias** [rubberduck-vba/Rubberduck#5956]
**Would it be beneficial to do a refresh whenever opening a RD window?**

I think my workflow would benefit from it, but I'm not sure if this holds in general.

So here's my workflow:
I keep open certain RD windows all the time and I'm used to do a refresh whenever
necessary. But then there are some windows that I open just at the moment
I really need them.

For example, I keep the Test Explorer closed most of the time. And create, rename,
and remove Test Methods while the Test Explorer is closed. Finally I open the Test Explorer
to run my tests. But all too often I forget to do a refresh before running the test which 
is confusing: Tests that I renamed are showing up with their old names, and tests that I
removed are still visible in Test Explorer. Thus in my workflow I always need to do a refresh after 
opening Test Explorer. And it would be great if a refresh happened automatically every
time I open Test Explorer.

I'm not too sure if all RD windows would benefit from an automatic refresh during opening.
My workflow would not be slowed down by an automatic refresh but it might be different
for someone opening and closing RD windows continuously.

**Comment by retailcoder** [comment]
Nah, it's part of what differentiates Rubberduck from others: all toolwindows always reflect the current state of the add-in's understanding of the code, and triggering a parse has implications with the startup code that might destabilize it; best leave everything around startup alone.
